inspirehep/dojson/hep/model.py

```diff
@@ -53,9 +57,26 @@ def add_book_info(record, blob):
     return record
 
 
+def add_inspire_category_from_arxiv_categories(record, blob):
+    if not record.get('arxiv_eprints') or record.get('inspire_categories'):
+        return record
+
+    record.setdefault('inspire_categories', [])
+    for arxiv_category in record.get('arxiv_eprints').get('categories'):
```

Comment:
I will change it. Anyway `get_value` it's 5 times slower in according to the Jan's comment (https://github.com/inspirehep/inspire-next/blob/master/inspirehep/utils/record.py#L62).